Adyen/UI/Form/Items/Value Pickers/Identifier Picker/FormStringPickerItem.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+/// A wrapper struct to use as item in ``FormStringPickerItem``
+@_spi(AdyenInternal)
+public struct FormStringPickerElement: CustomStringConvertible, Equatable {
+
+    public let identifier: String
+    public let title: String
+    public var description: String {
+        title
+    }
+
+    public init(identifier: String, title: String) {
+        self.identifier = identifier
+        self.title = title
+    }
+}
+
+/// A identifier picker form item
+@_spi(AdyenInternal)
+public final class FormStringPickerItem: BaseFormPickerItem<FormStringPickerElement> {
+
+    public init(
+        preselectedIdentifier: FormStringPickerElement,
```

Comment:
parameter names still have identifier. Can rename to `preselectedValue` or `preselectedStringValue`

Reply:
Renamed them :)